app/app.go

```diff
@@ -203,12 +235,20 @@ func (a *App) PushHandler(push *msg.Push) {
 	}
 }
 
-// getLocalChain returns an instance of the blockchain.
-func getLocalChain() *blockchain.BlockChain {
-	// TODO: Look for local chain on disk. If doesn't exist, go rummaging
+// createBlockchain returns a new instance of a blockchain with only a genesis
+// block.
+func createBlockchain(user *User) *blockchain.BlockChain {
```

Comment:
Is this function just for testing? Why is the current user creating the genesis block in this block chain? `getLocalChain` actually performed a function that we can improve upon, `createBlockchain` seems to deliberately create an invalid blockchain.